compiler-rt/cmake/Modules/AllSupportedArchDefs.cmake

```diff
@@ -32,6 +32,9 @@ set(ALL_ASAN_SUPPORTED_ARCH ${X86} ${X86_64} ${ARM32} ${ARM64} ${RISCV64}
     ${LOONGARCH64})
 set(ALL_ASAN_ABI_SUPPORTED_ARCH ${X86_64} ${ARM64} ${ARM64_32})
 set(ALL_DFSAN_SUPPORTED_ARCH ${X86_64} ${MIPS64} ${ARM64} ${LOONGARCH64})
+set(ALL_RADSAN_SUPPORTED_ARCH ${X86} ${X86_64} ${ARM32} ${ARM64} ${RISCV64}
```

Comment:
Just a reminder - let's make a note of questions to ask the reviewers at the top of our PR - any preference on where we keep this list?

Reply:
OK great. Good idea on the Google doc - I've created one [here](https://docs.google.com/document/d/1q197se0-q3Pt8Lf3e_i2g9oqCHiaa-yDCdhwYk7CibM/edit?usp=sharing)